common/web/input-processor/src/tsconfig.bundled.json

```
@@ -5,7 +5,8 @@
   // for certain down-line builds if the two tsconfigs are not differentiated.
   "extends": "./tsconfig.json",
   "compilerOptions": {
-    "outFile": "../build/index.bundled.js"
+    "outFile": "../build/index.bundled.js" //,
+    //"declaration": false,
```

```suggestion
    "outFile": "../build/index.bundled.js",
```

Note that the trailing comma is permitted in JSON-with-comments, so I left it in as it makes future changesets cleaner